packages/slate-editor/src/modules/editor/lib/isEditorValueEqual.ts

```diff
@@ -13,7 +18,7 @@ export function isEditorValueEqual<T extends Descendant>(editor: Editor, a: T[],
         if (!isNodeText && !isAnotherText) {
             const equal = editor.isElementEqual(node as Element, another as Element);
             if (typeof equal !== 'undefined') {
-                if (editor.isVoid(node) || editor.isVoid(another)) {
```

Comment:
I think my only answer would be that it's easier to use `editor.{method}` because you don't have to import extra stuff?

Or, in case the `editor.{method}` does not exist, you import the specific function.

I don't remember exactly why I did it this way, but I'd say it's better to go with the `editor.{method}` approach for less overal imports.